lib/ReactViews/SelectableDimensions/Color.tsx

```diff
@@ -23,7 +23,7 @@ const debounceSetColorDimensionValue = debounce(
 export const SelectableDimensionColor: React.FC<{
   id: string;
   dim: SelectableDimensionColorModel;
-}> = observer(({ id, dim }) => {
+}> = observer(({ dim }) => {
```

Comment:
I was unsure of side effects here, whether removing `id` would make the observer lose sensitivity to changes to `id`